recipes-graphics/mesa/mesa-gl_%.bbappend

```diff
@@ -2,5 +2,5 @@ PACKAGECONFIG_append_rpi = " gbm gallium"
 PROVIDES_append_rpi = " virtual/libgbm"
 
 do_install_append_rpi() {
-    rm -rf ${D}${includedir}/KHR/khrplatform.h
+    rm ${D}${includedir}/KHR/khrplatform.h
```

Comment:
what do we get by doing this ?

Reply:
```
       -f, --force
              ignore nonexistent files and arguments
       -r, -R, --recursive
              remove directories and their contents recursively
```
this is a file so `-r` don't make sense
without `-f` this will fail the install task when the file don't exist and this will signal that this line can be removed

Reply:
I think I know what these options mean, that's not the question. why do we want to fail when file does not exist?

Reply:
Sorry if i didn't explain myself well.

If the file does not exist the `rm` command doesn't make sense.
and with this pacth this situation can be checked if for some reason this file are renamed, moved or removed in the future.
fail the install task will show that the file doesn't exist and this `rm` can be removed in this situation.

In real this is not needed in this PR and don't fixe anything but my intention with this is mainly for make the `rm` command to fail when something change in future.

`rm -rf ${D}${includedir}/KHR/khrplatform.h` will always return 0 even if this file disappear for some reason

Reply:
but thats the point of using `-f` we don't want to have this file at all so if it exists or not does not matter. I am fine with removing `-r`

Reply:
Your PR https://github.com/agherzan/meta-raspberrypi/pull/774 fixes the problems that I have with `mesa-gl`, so I will drop the `mesa-gl` pacthes from this PR